src/sonic-host-services/scripts/hostcfgd

```diff
@@ -109,6 +130,22 @@ class Feature(object):
             raise ValueError('Invalid state rendered for feature {}: {}'.format(self.name, target_state))
         return target_state
 
+    def __safe_eval_bool(self, val):
+        """ Safely evaluate the boolean expression, without raising an exception """
+        try:
+            ret = ast.literal_eval(val)
+        except ValueError:
```

Comment:
>ValueError [](http://example.com/codeflow?start=15&length=10)

Could you give input examples which throw exception? Add test cases?

Reply:
You may add some code comment or add test. My understanding is that it is not limited to boolean value, is it true?